recordset/recordset.app.js

```diff
@@ -64,7 +64,11 @@
         selectedRows: [],   // array of selected rows
         search: null,       // search term
         pageLimit: 25,      // number of rows per page
-        config: {}
+        config: {},
+        tooltip: {
+            actionCol: "Click on the action buttons to view, edit, or delete each record",
+            viewCol: "Click on the eye button to view the detailed page associated with each record"
```

Comment:
Both of these tooltips should be on the message map object. The "column" that these tooltips are being added to are part of the table directive which can appear in all 3 apps, so it doesn't make sense to add it here to the `recordsetModel`.